Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.7.5-wheezy
+
+COPY . /home
+
+WORKDIR /home
+
+# supervisord.conf contains configuration for controlling Minio and worker processes.
+COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+
+RUN \
```

Comment:
Always combine multiple RUNs together in docker, to avoid unnecessary layers being created.